### PR TITLE
Fix issues with Godeps.json

### PR DESCRIPTION
**Pull request by ncdc**
- Remove duplicate entries
- Correct "Comment" for github.com/containernetworking/cni/...
- Correct "Comment" and "Rev" for github.com/opencontainers/runc

@soltysh @deads2k @liggitt @smarterclayton

### CODE REVIEW

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -391,43 +391,33 @@
...
-		},
-		{
-			"ImportPath": "github.com/containernetworking/cni/pkg/invoke",
-			"Comment": "v0.2.3",
-			"Rev": "52e4358cbd540cc31f72ea5e0bd4762c98011b84"
-		},
-		{
 			"ImportPath": "github.com/containernetworking/cni/pkg/ip",
-			"Comment": "v0.2.3",
+			"Comment": "v0.3.0-83-g52e4358",
```
Comment (deads2k) [inline Godeps/Godeps.json]: found it.  guess so.

**Comment by deads2k** [comment]
lgtm.  want to force it ahead of your godep.json only bump of kube levels?

File: Godeps/Godeps.json (hunk 1 above)
Comment (ncdc) [inline Godeps/Godeps.json]: I didn't check the history of what it's out of sync, but it's definitely cni commit 52e4358.

**Comment by deads2k** [comment]
We should probably write a tool that updates the comment to match too.  This is good enough.

**Comment by deads2k** [comment]
Checked it locally. Manually merging to add the marker.

**Pull request merged**